runtime/calamari/src/weights/pallet_timestamp.rs

```diff
@@ -54,24 +54,24 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_timestamp::WeightInfo for SubstrateWeight<T> {
     // Storage: Timestamp Now (r:1 w:1)
     fn set() -> Weight {
-        (7_882_000 as Weight)
+        (18_208_000 as Weight)
```

Comment:
2.5x increase in pallet_timestamp

Reply:
![image](https://user-images.githubusercontent.com/4237655/192064154-76d4cd20-0b4e-41f7-860d-e5e4f9852bd8.png)

benchmarking outlier. Only executed once per block. Ignoring